src/views/transactions/components/transaction-overview/transaction-overview.view.tsx

```diff
@@ -48,6 +43,7 @@ type Transaction =
       amount: BigNumber;
       account: Account;
       fee: number;
+      estimatedWithdrawFeeTask: AsyncTask<EstimatedWithdrawFee, Error>;
```

Comment:
You've moved `EstimatedWithdrawFee` to the domain but forgot to import it in this module, so `EstimatedWithdrawFee` is `any` here.

You get no compile errors in this file because both TS and eslint are disabled. Comment lines 2 and 3  to see all errors.

Reply:
Ahhhh didn't see that comments! Done.

Reply:
Yeah, disabling TS and the linter is problematic, I've also been bugged by these comments, but we need them until we can migrate the components.